### PR TITLE
Adapt to core and plugin changes

### PR DESCRIPTION
**Pull request by timja**
Adapt to:

- https://github.com/jenkinsci/jenkins/pull/6455
- https://github.com/jenkinsci/jenkins/pull/6464

TODO:
- Regression in https://github.com/jenkinsci/credentials-plugin/pull/296 needs fixing, see https://github.com/jenkinsci/acceptance-test-harness/pull/759#issuecomment-1103226304

FYI @NotMyFault @janfaracik 

Let's get these builds green again before testing anything

### CODE REVIEW

File: src/main/java/org/jenkinsci/test/acceptance/plugins/ssh_slaves/SshSlaveLauncher.java [hunk 1]
```diff
@@ -16,7 +16,7 @@
 /**
  * @author Kohsuke Kawaguchi
  */
-@Describable("hudson.plugins.sshslaves.SSHLauncher")
+@Describable({"Launch agents via SSH", "hudson.plugins.sshslaves.SSHLauncher"})
```
Comment (jtnord) [inline src/main/java/org/jenkinsci/test/acceptance/plugins/ssh_slaves/SshSlaveLauncher.java:19]: this is likely going to be an issue as there are multiple ways to "launch agents via SSH" (one is cloudbees specific with non blocking, and the other is the OSS ssh-slaves plugin.  we should always be able to use the class name of the describable to not be dependant on any naming changes.  has the UI changed such that the describable is no longer picked up correctly (should be ok as without it databinding will likely not work)

**Comment by jtnord** [comment]
the majority of remaining failures seem to be the inability to select credentials for the SSH agents

```
Unable to locate By.xpath: //*[@id='credentials-dialog-form']//*[@path='/'] in http://127.0.0.1:62883/computer/createItem
Stacktrace
org.openqa.selenium.NoSuchElementException: Unable to locate By.xpath: //*[@id='credentials-dialog-form']//*[@path='/'] in http://127.0.0.1:62883/computer/createItem
	at org.jenkinsci.test.acceptance.po.CapybaraPortingLayerImpl.find(CapybaraPortingLayerImpl.java:199)
	at org.jenkinsci.test.acceptance.po.Control.resolve(Control.java:75)
	at org.jenkinsci.test.acceptance.po.Control.select(Control.java:286)
	at org.jenkinsci.test.acceptance.plugins.ssh_credentials.SshCredentialDialog$1.resolve(SshCredentialDialog.java:36)
	at org.jenkinsci.test.acceptance.po.CapybaraPortingLayerImpl$Resolver.find(CapybaraPortingLayerImpl.java:532)
	at org.jenkinsci.test.acceptance.po.CapybaraPortingLayerImpl.findCaption(CapybaraPortingLayerImpl.java:483)
	at org.jenkinsci.test.acceptance.plugins.ssh_credentials.SshCredentialDialog.select(SshCredentialDialog.java:33)
```

File: src/main/java/org/jenkinsci/test/acceptance/plugins/ssh_slaves/SshSlaveLauncher.java (hunk 1 above)
Comment (timja) [reply src/main/java/org/jenkinsci/test/acceptance/plugins/ssh_slaves/SshSlaveLauncher.java:19]: It's changed to use https://github.com/jenkinsci/jenkins/pull/6464

Which is keyed by the loop index, https://github.com/jenkinsci/jenkins/blob/master/core/src/main/resources/lib/form/dropdownDescriptorSelector.jelly#L61

**Comment by timja** [comment]
Credentials plugin issue caused by https://github.com/jenkinsci/jenkins/pull/6464

**Comment by timja** [comment]
The other test failing is:
`plugins.NodeLabelParameterPluginTest#run_on_several_online_and_offline_slaves`

Bisect shows:
```
900374e12421204f9ad76ac84397e96b9399cf00 is the first bad commit
commit 900374e12421204f9ad76ac84397e96b9399cf00
Author: Jan Faracik <43062514+janfaracik@users.noreply.github.com>
Date:   Fri Apr 15 21:59:48 2022 +0100

    [JENKINS-68129] Fix dropdown chevrons not separating links and overflowing (#6481)

 war/src/main/less/modules/breadcrumbs.less | 175 ++++++++++++++++-------------
 war/src/main/less/modules/page-header.less |  13 ++-
 war/src/main/less/modules/table.less       |   6 +
 3 files changed, 113 insertions(+), 81 deletions(-)
```

cc @janfaracik 

haven't confirmed yet if regression or test needs changing

**Comment by NotMyFault** [comment]
The checks are passing here but failing again on master, huh?

**Comment by timja** [comment]
> The checks are passing here but failing again on master, huh?

yes because https://github.com/jenkinsci/jenkins/pull/6519 is not released yet.

I used the incremental from master branch to test this in the meantime

**Pull request merged**